go.mod

```diff
@@ -150,11 +150,11 @@ require (
 	github.com/yusufpapurcu/wmi v1.2.3 // indirect
 	go.mongodb.org/mongo-driver v1.11.0 // indirect
 	go.opencensus.io v0.24.0 // indirect
-	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.48.0 // indirect
-	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.48.0 // indirect
-	go.opentelemetry.io/otel v1.23.0 // indirect
-	go.opentelemetry.io/otel/metric v1.23.0 // indirect
-	go.opentelemetry.io/otel/trace v1.23.0 // indirect
+	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.49.0 // indirect
+	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.49.0 // indirect
```

Comment:
because i think it will be overwritten with any `go mod tidy`
`go mod why -m go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc` 
```
# go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc
github.com/weaviate/weaviate/modules/backup-gcs
cloud.google.com/go/storage
cloud.google.com/go/storage/internal/apiv2
google.golang.org/api/transport/grpc
go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc
```

Reply:
or we can have `replace` to make sure it's not overwritten

Reply:
just to be on the safe zone with fresh containers to pick the version we would like

Reply:
Updated by adding otel in the require list as direct dependency.